src/popup/components/Modals/MultisigVaultCreate.vue

```diff
@@ -6,126 +6,153 @@
     class="multisig-vault-create"
     @close="reject()"
   >
-    <MultisigVaultCreationProgress
-      v-if="showVaultCreationProgress"
-      :progress="multisigProgress"
-      :multisig-account="multisigAccount"
-    />
-    <template v-else>
-      <div class="content">
-        <h2 class="title">
-          {{ $t('modals.createMultisigAccount.title') }}
-        </h2>
-        <div
-          v-for="(signer, index) in signers"
-          :key="`signer${index}`"
+    <!--
+      Step 1: The form
+    -->
+    <form
+      v-if="currentStep === STEPS.form"
+      class="multisig-vault-create-form"
+      @submit.prevent="openReviewStep"
+    >
+      <h2 class="text-heading-1">
+        {{ $t('modals.createMultisigAccount.title') }}
+      </h2>
+      <div
+        v-for="(signer, index) in signers"
+        :key="`signer${index}`"
+      >
+        <InputField
+          v-model.trim="signer.address"
+          v-validate="{
+            required: true,
+            name_registered_address_or_url: true,
+          }"
+          :name="`signer-address-${index}`"
+          :label="getSignerLabel(index)"
+          :placeholder="$t('modals.createMultisigAccount.signerInputPlaceholder')"
+          :message="errors.first(`signer-address-${index}`)"
```

Comment:
I don't like that you are using almost the same template string three times here - maybe it should be wrapped in a function to return `signer-address-${index}`?

Reply:
This is used twice. And it's normal to do it this way when binding the v-validate rules with collecting the errors for dynamic fields.